### PR TITLE
Migrate Big Query dialect implementation from the virtual-schemaS repository

### LINKED ISSUE #1: Migrate BigQuery dialect implementation from the virtual-schemas repository
Migrate BigQuery dialect implementation from the virtual-schemas repository

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
Closes #1.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-assembly-plugin</artifactId>
+				<version>3.3.0</version>
+				<configuration>
+					<descriptors>
+						<descriptor>src/assembly/all-dependencies.xml</descriptor>
+					</descriptors>
+					<finalName>virtual-schema-dist-${vscjdbc.version}-db2-${version}</finalName>
```
Comment (AnastasiiaSergienko) [inline pom.xml:130]:
```suggestion
					<finalName>virtual-schema-dist-${vscjdbc.version}-bigquery-${version}</finalName>
```

File: doc/changes/changes_1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Big Query Virtual Schema 1.0.0, released 2021-01-15
+
+Code name: Migration from the virtual-schemas repository
+
+## Features / Enhancements
+
+* 1: Migrated from the virtual-schemas repository
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_1.0.0.md:7]:
```suggestion
* #1: Migrated from the virtual-schemas repository 
```

File: doc/user_guide/bigquery_user_guide.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+FLOAT              |  ✓        | DOUBLE                    | Expected range for correct mapping: -99999999.99999999 .. 99999999.99999999. 
+GEOGRAPHY          |  ✓        | VARCHAR(65535)            |
+INTEGER            |  ✓        | DECIMAL(19,0)             | 
+NUMERIC            |  ✓        | VARCHAR(2000000)          | 
+RECORD/STRUCT      |  ×        |                           | 
+STRING             |  ✓        | VARCHAR(65535)            | 
+TIME               |  ✓        | VARCHAR(16)               | 
+TIMESTAMP          |  ✓        | TIMESTAMP                 | Expected range for correct mapping: 1582-10-15 00:00:01 .. 9999-12-31 23:59:59.9999. JDBC driver maps dates before 1582-10-15 00:00:01 incorrectly.  Example of incorrect mapping: 1582-10-14 22:00:01 -> 1582-10-04 22:00:01
+
+If you need to use currently unsupported data types or find a way around known limitations, please, create a github issue in the [VS repository](https://github.com/exasol/virtual-schemas/issues).
```
Comment (AnastasiiaSergienko) [inline doc/user_guide/bigquery_user_guide.md:96]:
```suggestion
If you need to use currently unsupported data types or find a way around known limitations, please, create a github issue in the [VS repository](https://github.com/exasol/bigquery-virtual-schema/issues).
```

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]

**Review by AnastasiiaSergienko — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+* [Virtual Schema API Documentation][vs-api]
+
+### Run Time Dependencies
+
+Running the Virtual Schema requires a Java Runtime version 11 or later.
+
+| Dependency                                                         | Purpose                                                | License                           |
+|--------------------------------------------------------------------|--------------------------------------------------------|-----------------------------------|
+| [Exasol Virtual Schema JDBC][virtual-schema-common-jdbc]           | Common JDBC functions for Virtual Schemas adapters     | MIT License                       |
+| [Exasol Error Reporting][exasol-error-reporting]                   | Creating unified error messages                        | MIT License                       |
```
Comment (AnastasiiaSergienko) [inline README.md:50]: JDBC driver is missing here?

Comment (chiaradiamarcelo) [reply README.md]: fixed

**Pull request merged**